server/src/schemas/typeDefs.ts

```diff
@@ -40,18 +45,18 @@ const typeDefs = `
   type Query {
     users: [User]
     user(username: String!): User
-    thoughts: [Thought]!
-    thought(thoughtId: ID!): Thought
+    recipes: [Recipes]!
+    recipeName: [Recipes]
```

Comment:
`recipe(recipeId: ID!): Recipe`
Think this is how we might need to pull single recipes, since we need ot take in some kind of argument that identifies which recipe we return.